### PR TITLE
Dialog qa updates

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

File: packages/core/stories/dialog/dialog.qa.stories.tsx [hunk 1]
```diff
@@ -4,18 +4,23 @@ import {
...
+  VALIDATION_NAMED_STATUS,
 } from "@salt-ds/core";
 import type { Meta, StoryFn } from "@storybook/react";
 import { QAContainer, type QAContainerProps } from "docs/components";
-
 import "./dialog.stories.css";
+import { Fragment } from "react";
+
+export default {
+  title: "Core/Dialog/QA",
```
Comment (joshwooding) [inline packages/core/stories/dialog/dialog.qa.stories.tsx:21]:
```suggestion
  title: "Core/Dialog/Dialog QA",
```

File: packages/core/stories/dialog/dialog.qa.stories.tsx [hunk 2]
```diff
@@ -25,256 +30,220 @@ function FakeDialog({ children, status, id }: DialogProps) {
...
+        themeNext: "enable",
+        corner: "rounded",
+        accent: "teal",
+        // Ignore headingFont given font is not loaded
+      },
+    },
+  },
+};
+
+export const ContentVariants: StoryFn<QAContainerProps> = () => {
```
Comment (origami-z) [inline packages/core/stories/dialog/dialog.qa.stories.tsx:114]: Looking at Chromatic, there are some short content with blank spacing? Is it intended usage?

**Review by origami-z — COMMENTED** [review]
👍 for consolidation

File: packages/core/stories/dialog/dialog.qa.stories.tsx (hunk 2 above)
Comment (Fercas123) [reply packages/core/stories/dialog/dialog.qa.stories.tsx:114]: yes, it is showing sticky footer :)

**Review by origami-z — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**